src/components/App/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import './App.css';
+import WebAudioEngine from 'components/WebAudioEngine';
+
+class App extends Component {
+
+  render() {
+    return (
+      <div className="App">
+        <p>
+          <h2>React Web Audio Demo</h2>
```

Comment:
Getting console message: `<h2> cannot appear as a descendant of <p>`
Remove p tags around h2 to fix